data/ecosystems/e/ethereum.toml

```diff
@@ -6794,6 +6796,9 @@ url = "https://github.com/LaTribuWeb3/dex-history-fetcher"
 [[repo]]
 url = "https://github.com/LaTribuWeb3/fei-protocol-core"
 
+[[repo]]
+url = "https://github.com/atomizexyz/fenix"
```

Comment:
you can remove this repo, since it's already in the Ethereum's child "Fenix" ecosystem